packages/server/prisma/schema.prisma

```diff
@@ -0,0 +1,42 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Post {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  title     String   @db.VarChar(255)
+  content   String?
+  published Boolean  @default(false)
+  author    User     @relation(fields: [authorId], references: [id])
+  authorId  Int
+}
+
+model Profile {
+  id     Int     @id @default(autoincrement())
+  bio    String?
+  user   User    @relation(fields: [userId], references: [id])
+  userId Int     @unique
+}
+
+model User {
+  id      Int      @id @default(autoincrement())
+  email   String   @unique
```

Comment:
we may also need an attribute `isFirstLogin: boolean` for landing page ❓

Reply:
These models are here only for example. We ca add/remove these however we like in the future.

Reply:
Having these example models will already cause migrations in the database (even when setting up the production database). Not sure if is a good thing... 🤔 Maybe we should already look into designing some first real tables and use those here instead of examples.

Reply:
Maybe even just a few fields for a first table at this point, just to get some real fields for now, but without having to design the whole thing blocking this merge.